.changeset/README.md

```diff
@@ -0,0 +1,8 @@
+# Changesets
```

Comment:
does changesets read package.json workspaces automatically to determine what a package is and what is not? I'm pretty sure it does, would only like to reassure myself

Reply:
Correct, It uses your `workspaces` field to determine what is and is not a package.

If you run `yarn changeset` it will also show you a list of all the things it has detected as packages if you want to verify what it's affecting. I'll make a note to clarify this in the docs.